core-api/src/main/java/com/optimizely/ab/optimizelyconfig/OptimizelyConfigService.java

```diff
@@ -31,6 +33,8 @@ public class OptimizelyConfigService {
     private Map<String, List<FeatureVariable>> featureIdToVariablesMap = new HashMap<>();
     private Map<String, OptimizelyExperiment> experimentMapByExperimentId = new HashMap<>();
 
+    private static final Logger logger = LoggerFactory.getLogger(OptimizelyConfigService.class);
+
     public OptimizelyConfigService(ProjectConfig projectConfig) {
```

Comment:
Is it worth adding an overload/optional parameter to the constructor to allow logger to be created from outside?  🤔

Reply:
java-sdk uses SFL4J and creates a logger for each class. It may be worth, but we do not follow the logger injection pattern here.